src/authors-details.js

```diff
@@ -0,0 +1,19 @@
+//TODO: data related to each author on the blog posts should be added incrementally here.
+
+module.exports = [{
+  name: "Joel Coutinho",
+  jobTitle: "Software Developer @Supertokens",
```

Comment:
The @SuperTokens should link to our twitter? Or then remove the @..

Reply:
it does not like to our twitter handle. removed @...